xdsl/dialects/hw.py

```diff
@@ -409,12 +426,437 @@ def print_parameters(self, printer: Printer):
             printer.print_string("]")
 
 
+class Direction(Enum):
+    """
+    Represents the direction of a module port.
+    """
+
+    # TODO: support INOUT direction
```

Comment:
Can you file an issue please and add it to the comment

Reply:
#2368